Add isolayer selection method (#3846)
MDAnalysis/mdanalysis: package/doc/sphinx/source/documentation_pages/selections.rst

    resid 130 or resid 80 ) )`` selects the center of geometry of protein,
    resid 130, resid 80 and creates a sphere of radius 6.0 around the COG.

isolayer *inner radius* *outer radius* *selection*
    Similar to sphlayer, but will find layer around all referenced atoms. 
    For example, if the atom types for a polymer backbone were chosen, then
    an isolayer parallel to the backbone will be generated. As another 

>     example, if an ions were chosen as a reference to isolate the second 

Reviewer: Did you mean a single or multiple ions?
```suggestion
    example, if an ion was chosen as a reference to isolate the second 

```

Author: I meant multiple. I've pushed the changes.

>     hydration layer, then those solvent atoms that overlap will not include

Reviewer: I'm not sure I fully follow the use of the word "overlapping" here? Could you clarify for me possibly?

Author: I think I clarified it adequately in my latest push.
